.github/DISCUSSION_TEMPLATE/ideas.yml

```
@@ -0,0 +1,43 @@
+title: "(packageName) Summary of feature or enhancement"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for wishing to improve packages that we host in this repository.
+
+        Before you fill out the template here, please make sure that the title contains a very small summary of the the new enhancement or feature, that the title starts with `(packageName)`, and that can be give an idea what the discussion is about.
+        **NOTE: Replace `packageName` with the name of the package**
+  - type: checkboxes
+    attributes:
+      label: Before starting, ensure that you have done the following.
+      options:
+        - label: I have verified that there is no other issue or discussion covering the feature or enhancement
+          required: true
+        - label: I have verified there is no open pull request for the feature or enhancement
+          required: true
+        - label: I have verified the package to improve is located in this repository
+          required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: What do you want to add?
+      description: Please explain what the feature or enhancement is about, and to explain it in a way you would normally explain it to people if you had to convince them of why it would be a benefit to add this feature or enhancement.
+    validations:
+      required: true
+  - type: textarea
+    id: problem-related
+    attributes:
+      label: What problem will the feature/enhancement solve?
+      description: |
+        If the feature or enhancement is added, what problem that you are seeing will this improvement solve?
```

It feels like this is clearer if we just ask what the problem is, not necessarily what they're seeing?

```suggestion
        If the feature or enhancement is added, what problem will this improvement solve?
```